infrastructure/resource_group_level/main.bicep

```diff
@@ -189,9 +191,10 @@ var tfmApiAdditionalSecureConnectionStrings = {
 }
 
 var portalUiSettings = {
-    RATE_LIMIT_THRESHOLD: RATE_LIMIT_THRESHOLD // TODO:FN-1086 30 on dev, 10000 on feature
-    COMPANIES_HOUSE_API_URL: COMPANIES_HOUSE_API_URL
-    MAX_UTILISATION_REPORT_FILE_SIZE: MAX_UTILISATION_REPORT_FILE_SIZE
+  RATE_LIMIT_THRESHOLD: RATE_LIMIT_THRESHOLD // TODO:FN-1086 30 on dev, 10000 on feature
+  COMPANIES_HOUSE_API_URL: COMPANIES_HOUSE_API_URL
+  MAX_UTILISATION_REPORT_FILE_SIZE: MAX_UTILISATION_REPORT_FILE_SIZE
+  PORTAL_UI_URL: PORTAL_UI_URL
```

Comment:
`portal-api` is the service that needs to know this info, not `portal-ui`, so I'm guessing it should be in `portalApiSecureSettings` instead